### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by drashbot**
This was auto-generated by GitHub Actions.

### CODE REVIEW

**Comment by ebebbington** [comment]
is this something we want to do at the moment?

**Comment by crookse** [comment]
yea but let's also fix our implementations in this PR

**Comment by ebebbington** [comment]
oh yeah they moved the websocket stuff, ill look into it now

**Comment by ebebbington** [comment]
**KEY CHANGES**

I felt having a web server was kind of unecessary, as we want to make sure that our socket server can get and receieve messages, whereas before, it was only testing getting messages. So i removed the web server and simplified the tests (using std/ws tests as a guideline too)

Alongside this, the data sent to the ws server was actually incorrect - it was an object like `{ data: { connect_to: ... }}` as the main parent prop, when it shouldn't have the data prop (or at least the server doesn't expect it to)

**Comment by ebebbington** [comment]
And added extra tests

File: tests/integration/app_3000/tests.ts [hunk 1]
```diff
@@ -101,16 +41,136 @@ console.log(
...
+    //   WSClient1.onmessage = function (message) {
+    //     console.log('ws client 1 got message')
+    //     console.log(message)
+    //     WSClient1.close()
+    //   }
+    //   WSClient1.onclose = function () {
+    //     promise.resolve();
+    //   };
+    //   await promise;
+    // })
```
Comment (ebebbington) [inline tests/integration/app_3000/tests.ts:133]: unsure how to test this though

**Comment by ebebbington** [comment]
ready for review

**Review by crookse — APPROVED** [review]
LGTM

**Pull request merged**